lightbend-telemetry/log-correlation-java-mvn/README.md

```diff
@@ -0,0 +1,46 @@
+# This is an example how to use Lightbend Telemetry (Cinnamon) for Log Correlation
+
+How it work:
+
+Cinnamon starting 2.10.2 support auto log correlation generation.
+
+In order to enable it use next configuration:
+
+```
+cinnamon.slf4j.mdc {
+  correlation-id {
+    automatic = on
+    name = "X_CORRELATION_ID"
+  }
+}
+``` 
+
+How to run:
+
+```
+mvn install                  # builds the service and downloads the Cinnamon agent
+mvn -pl hello-impl exec:exec # runs the hello service
+```
+
+Test:
```

Comment:
It'd be great if we had a little explanation of what these logs and `curl` statement demonstrate. I had to go check the implementation in `HelloServiceImpl` to confirm that there was a remote call involved (the fact that both methods are in `HelloServiceImpl` and on the same process didn't help).